app/Components/header/menu/Menu.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import s from '../headerStyle.module.css';
+
+const data = [
+    {
+        title: "Main",
+        href: '#Main'
+    },
+    {
+        title: "Invitation",
+        href: '#Invitation'
+
+    },
+    {
+        title: "Schedule",
+        href: '#Schedule'
+    },
+    {
+        title: "Contacts",
+        href: '#Contacts'
+    }
+]
+
+const Menu = () => {
+    return (
+        <ul className={s.ling_container}>
+            {data.map(item => {
+                return (
+                    <li key={item.title}>
+                        <a className={s.link} href={item.href}>
+                            {item.title}
+                        </a>
+                    </li>
+                )
+            })}
```

Comment:
If we don't have any logic it will be clearer if you remove return 
```suggestion
          {data.map(item => 
                  (<li key={item.title}>
                        <a className={s.link} href={item.href}>
                            {item.title}
                        </a>
                    </li>)
          )}
```

Reply:
this component is not used, I will remove it